### PR TITLE
DDT-888 refactored GlobalExceptionHandler to support chaining

### PR DESCRIPTION
**Pull request by pimg**
resolves: DDT-888

### CODE REVIEW

File: src/main/java/org/dcsa/core/exception/handler/GlobalExceptionHandler.java [hunk 1]
```diff
@@ -80,15 +84,31 @@
 
   @ResponseStatus(value = HttpStatus.BAD_REQUEST, reason = "invalidInput")
```
Comment (douradofrazer) [inline src/main/java/org/dcsa/core/exception/handler/GlobalExceptionHandler.java:85]: We don't need this anymore, from what I understand `handleConcreteRequestErrorMessageException` uses the value of the annotation at a class level for the custom exception throw.

File: src/main/java/org/dcsa/core/exception/handler/GlobalExceptionHandler.java [hunk 2]
```diff
@@ -134,53 +168,69 @@ public void handle(ServerWebInputException ex) {
   // considerably better message.
   @ResponseStatus(HttpStatus.BAD_REQUEST)
```
Comment (douradofrazer) [inline src/main/java/org/dcsa/core/exception/handler/GlobalExceptionHandler.java:169]: Ditto.

**Review by douradofrazer — CHANGES_REQUESTED** [review]

File: src/main/java/org/dcsa/core/exception/handler/GlobalExceptionHandler.java (hunk 1 above)
Comment (pimg) [reply src/main/java/org/dcsa/core/exception/handler/GlobalExceptionHandler.java]: removed

File: src/main/java/org/dcsa/core/exception/handler/GlobalExceptionHandler.java (hunk 2 above)
Comment (pimg) [reply src/main/java/org/dcsa/core/exception/handler/GlobalExceptionHandler.java]: removed

**Review by douradofrazer — APPROVED** [review]

**Pull request merged**